### ISSUE TITLE
Cannot play url ingress locally

### ISSUE THREAD

**Issue by serhiynovos** [livekit/ingress#299]
When I start ingress to play audio by local ip address it's not being played, I just wee EOS log

<img width="1723" alt="image" src="https://github.com/user-attachments/assets/de5ae94d-dcf0-4a1e-b781-81b1748965bf">

http://192.168.28.149/cms/public/uploads/audios/fa5d1cdc105793d43a64f71032ee6730d7.mp3

I tried to use remote host and http with the same file and it played well

**Comment by serhiynovos** [comment]
After some research I found that this issue not only with local urls but also can happen with remote URLs for audio and video but less frequently.

1.4.0 version is working fine and starting from 1.4.1 version I'm facing this issue. On 1.4.1 release notes I see this PR https://github.com/livekit/ingress/pull/250 which I feel can be root of cause

**Comment by Yohe-Am** [comment]
We're running into this as well. It seems not all files cause this issue. I'm not sure what exact property leads to the issue (it doesn't seem to be codecs) but I can share the `ffprobe` output if needed.

**Comment by serhiynovos** [comment]
> We're running into this as well. It seems not all files cause this issue. I'm not sure what exact property leads to the issue (it doesn't seem to be codecs) but I can share the `ffprobe` output if needed.

As I understand in 1.4.1 release they added asynchronous room joining which has some sync issue which means as let latency between ingress and http source than more frequently it happen. For me if http source and ingress are running in the same k8s cluster where latency is close to 0 it always happens. In other cases when try to use remote it's more stable but still is not working as solid as 1.4.0

**Comment by Yohe-Am** [comment]
Yeah, your diagnosis is correct. The issue completely disappears when I revert the changes made in the PR.

**Comment by Yohe-Am** [comment]
@ezBeanie go with version 1.4.0, it doesn't have the bug.

**Comment by serhiynovos** [comment]
> [@Yohe-Am](https://github.com/Yohe-Am) Thanks for this hint, we did infact revert to 1.4.0 but we dont want to be stuck at that version indefinitely.

Yeah it will be a huge problem in the future as at some period of time it it may stop support latest livekit server
